docs/remoteExams.md

```diff
@@ -113,7 +113,7 @@ Some notes about this configuration:
 - Students who are scheduled for a conflict exam will also be able to access the exam during the primary time slot. However, if they do so, they will be blocked from the exam during the conflict time slot.
 - After the timer expires the exam will auto-close and grade any saved but ungraded questions and show students their final score. After this time students will be unable to see any of the questions.
 - If a student closes their web browser before the exam is complete, their exam will be automatically closed and graded within 12 minutes after their timer expires. If they try and access their exam during this time it will immediately close and grade.
-- Before downloading final scores, wait at least 12 minutes after the last student would have finished to ensure all exams are closed. You can also check (and manually close exams) on the "Students" page under the assessment in PrairieLearn.
+- Before downloading final scores, wait at least 12 minutes after the last student would have finished (to ensure all exams are closed). You can also check (and manually close exams) on the "Students" page under the assessment in PrairieLearn.
```

Comment:
The 12 minute marker feels very magical -- why 12? Is there some other term we can refer to?

Reply:
I actually don't know where 12 minutes comes from. cc @mwest1066 who authored these docs. I see that https://github.com/PrairieLearn/PrairieLearn/issues/2241 asserts that 12 minutes is the value, but I don't understand why auto-close would happen then and not 6 hours later, which is what I've always told people.

Reply:
My guess is that this is based on the 10-minute default time limit for external graders (`config.externalGradingMaximumTimeout`), plus a couple of extra minutes for safety. When this was written all exams we didn't yet have `realTimeGrading=false` exams, so the auto-close time would not yet have been relevant.

Given the fact that we now have `realTimeGrading=false`, I think that 6 hours (plus a few minutes for external graders?) is the right number.

In general, I think we could reduce the auto-close time to something more like 1 hour, but that's outside the scope of this PR.

Reply:
This was split out into https://github.com/PrairieLearn/PrairieLearn/issues/11663. We can address this separately.